data/README.md

```diff
@@ -14,16 +16,17 @@ The Reverb platform generates an immense amount of log data that data engineers
 }
 ```
 
-In this example line, we can see that each log is identified with a unique `userId` and `sessionId`. A user can make multiple separate page views within the same session. We can also see that the `timestamp` of the page view is recorded along with the URL, as well as identifiers for any A/B tests the user may be a part of. Note that while the example here is broken out into multiple lines for readability, that in the file these logs are written 1 per line. You can assume that the timestamps increase monotonically.
+In this example, we can see that each event is identified with a unique `userId` and `sessionId`. A user can make multiple, separate page view events within the same session. We can also see that the `timestamp` of the page view is recorded along with the URL, as well as identifiers for any A/B tests the user may be a part of. _Note: While the example here is broken out into multiple lines for readability, event files include one event per line. You can assume that the timestamps increase monotonically._
 
 ## Instructions
 
-1. Familiarize yourself with the data. We've provided a Python skeleton app for ease, however you are encouraged to use the language and frameworks you are most comfortable with.
-2. If you bring your own computer to the interview, please ensure you are ready to go by having anything installed beforehand. If you want to use one of Reverb's computers, please tell us which code editor you prefer and which language you will be doing the extension in.
-3. That's it! We'll give you all other information during the coding session.
+1. Familiarize yourself with the [data](logs.txt).
+1. Familiarize yourself with the [app](python/app/) and [tests](python/test/). This is a Python skeleton app provided for ease, but you are encouraged alter it during the interview as you see fit. 
+1. That's it! We'll give you all other information during the coding session.
```

Comment:
```suggestion
2. Familiarize yourself with the [app](python/app/) and [tests](python/test/). This is a Python skeleton app provided for ease, but you are encouraged alter it during the interview as you see fit. 
3. That's it! We'll give you all other information during the coding session.
```

Reply:
That's an interesting part of markdown, it will automatically number when displaying. So by using the same number, you never have to worry when adding/removing/updating, etc.